tools.go

```diff
@@ -1,10 +1,11 @@
+//go:build tools
```

Comment:
needed ?
if so does the line below needed as well still ?

Reply:
This line is added by go, now that we use go 1.17 ...

Reply:
maybe need before that something like `go mod tidy -compat=$(GO_VERSION)` ?
weird that we have both lines now